cis194/week1/stevemao/src/Homework.hs

```diff
@@ -21,7 +21,7 @@ doubleOrNot a b
   | otherwise = a
 
 doubleEveryOther :: [Integer] -> [Integer]
-doubleEveryOther = mapInd doubleOrNot
+doubleEveryOther = reverse . mapInd doubleOrNot . reverse
```

Comment:
we don’t really care about the order of the digits before we sum them up.

Reply:
It says "Double the value of every second digit beginning from the *right*."
but `mapInd` is counting from the left.

Reply:
I was hoping you can figure out that the reverse left to mapInd is not needed.